vessim/core/consumer.py

```diff
@@ -62,7 +120,7 @@ class ComputingSystem(Consumer):
         pue: The power usage effectiveness of the system.
     """
 
-    def __init__(self, power_meters: List[PowerMeter], pue: float = 1):
+    def __init__(self, power_meters: list[PowerMeter], pue: float = 1):
```

Comment:
this is Python 3.9 code, but we also want to support 3.8

Reply:
Yes sry, I really have to configure my linter for that.